cmd/tink-worker/tink/tink.go

```diff
@@ -0,0 +1,70 @@
+package tink
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/pkg/errors"
+	pb "github.com/tinkerbell/tink/protos/workflow"
+)
+
+type Wdata struct {
+	Data WorkflowData
+}
+
+type WorkflowData struct {
+	WorkerID     string
+	WorkflowID   string
+	ActionStatus *pb.WorkflowActionStatus
+}
```

Comment:
These fields aren't shared across any of the methods you have defined. They don't feel cohesive at an individual method level. meaning, As a caller of `GetWorkflowContexts`, I don't think I should need to understand if I need `ActionStatus` or not. You might think about making these field values parameters to their respective methods. thoughts?